src/Component/FormControlled.js

```diff
@@ -0,0 +1,192 @@
+import React, {useState} from 'react'
+// import "./Form.css"
+import Alert from "./Alert";
+
+function FormControlled() {
+    const [state, setState] = useState({
```

Comment:
It's not that bad to use such a big object for useState, but I prefer using a useState for each prop because of all the rerenders that will be triggerd because it changes a lot.